GameSync.Api.IntegrationTests/EmailServiceTests.cs

```diff
@@ -0,0 +1,84 @@
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc.Testing;
+using GameSync.Api;
+using GameSync.Application.EmailInfrastructure;
+using Microsoft.Extensions.DependencyInjection;
+using NSubstitute;
+
+namespace MyProject.Api.IntegrationTests;
+
+
+public interface IEmailService
+{
+    Task SendEmailAsync(SendEmailCommand command);
+}
```

Comment:
This is a bit too far away in mocking and faking. You created the `fake` interface and then mocked it with `NSubsitutute`. You should mock the 'real' interface